app/src/main/java/com/framgia/wsm/screen/requestoff/RequestOffViewModel.java

```diff
@@ -1394,15 +1399,24 @@ public void onClickNext(View view) {
             mNavigator.startActivityForResult(ConfirmRequestOffActivity.class, bundle,
                     Constant.RequestCode.REQUEST_OFF);
         } else {
-            if (mEndDateHaveSalary == null || "".equals(mEndDateHaveSalary)) {
-                if (mEndDateNoSalary == null || "".equals(mEndDateNoSalary)) {
+            if (StringUtils.isBlank(mEndDateHaveSalary) && StringUtils.isBlank(
+                    mStartDateHaveSalary)) {
+                if (StringUtils.isBlank(mEndDateNoSalary) || StringUtils.isBlank(
+                        mStartDateNoSalary)) {
                     showErrorDialog(
                             mContext.getString(R.string.the_field_required_can_not_be_blank));
                     return;
+                } else {
+                    if (StringUtils.isNotBlank(mEndDateHaveSalary) || StringUtils.isNotBlank(
```

Comment:
ERROR: [CORRECTNESS][RpC_REPEATED_CONDITIONAL_TEST] Repeated conditional test in com.framgia.wsm.screen.requestoff.RequestOffViewModel.onClickNext(View)
http://findbugs.sourceforge.net/bugDescriptions.html#RpC_REPEATED_CONDITIONAL_TEST